viability/build.gradle

```diff
@@ -1,3 +1,8 @@
 import org.labkey.gradle.util.BuildUtils
 
-BuildUtils.addLabKeyDependency(project: project, config: "compile", depProjectPath: BuildUtils.getPlatformModuleProjectPath(project.gradle, "assay"), depProjectConfig: "apiCompile")
+dependencies{
```

Comment:
These don't actually have to be within the dependencies closure, since the addLabKeyDependency method accesses this closure to do its business.  There might be a slight configuration performance implication, but if so it's probably slight, so if making other modifications you could remove these, but no big deal.